build.xml

```diff
@@ -133,21 +135,21 @@
         <echo>Local github.sha: ${github.sha.local}</echo>
     </target>
 
-    <target name="get-local-git-branch" description="Retrieves the current git branch via command line.">
+    <target name="get-local-git-branch" description="Retrieves the current git branch via command  line.">
         <exec executable="git" outputproperty="github.branch.local">
             <arg value="branch"/>
             <arg value="--show-current"/>
         </exec>
         <echo>Local github.branch: ${github.branch.local}</echo>
     </target>
 
-    <target name="reset" description="Resets the build environment. Same as `clean`, but additionally deletes the `lib` directory with the Saxon and Xerces jar files.">
+    <target name="reset" description="Resets the build environment. Same as `clean`, but additionally deletes the `lib` directory with the TEI Stylesheets and the Saxon and Xerces jar files.">
```

Comment:
Please change the description of the target in the table with the targets in BUILD_COMMANDLINE accordingly